resources/debugger/py_debug_wrapper.py

```diff
@@ -0,0 +1,14 @@
+import os
+import runpy
+import sys
+
+if __name__ == '__main__':
+    # debugger will be available in the task root if we patched the user's requirements.txt correctly
+    task_root = os.environ['LAMBDA_TASK_ROOT']
+    print('Prepending Lambda task root to path: ' + task_root)
+    sys.path.insert(0, task_root)
+    print('Starting debugger')
```

Comment:
Introduced a replacement wrapper file that is handler agnostic so that we still have a string to look for before starting the debugger countdown